docs/fast_data/faq/debezium.md

```diff
@@ -0,0 +1,133 @@
+---
+id: debezium
+title: Debezium
+sidebar_label: Debezium
+---
+
+This page contains some useful answers to common questions about Debezium.
+
+# Debezium Connect
+This section covers the usage of Debezium as a Kafka Connect connector, specifically when using the official Docker image `quay.io/debezium/connect`.
+
+## How to filter messages before publishing to Kafka?
```

Comment:
This section can be useful also for [our Debezium Server configuration](https://docs.mia-platform.eu/docs/fast_data/connectors/debezium_cdc), cause from debezium doc it seems this also apply for debezium server.

It could be useful to move this section there and, here, put a link to it.

Reply:
I have created a dedicated section for Debezium Connect as an alternative of Debezium Server that do not requires Redis as a dependency